sdk/synapse/synapse-monitoring/review/synapse-monitoring.api.md

```diff
@@ -0,0 +1,116 @@
+## API Report File for "@azure/synapse-monitoring"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import * as coreHttp from '@azure/core-http';
+
+// @public (undocumented)
+export class MonitoringClient extends MonitoringClientContext {
+    constructor(credentials: coreHttp.TokenCredential | coreHttp.ServiceClientCredentials, endpoint: string, options?: MonitoringClientOptionalParams);
+    // Warning: (ae-forgotten-export) The symbol "Monitoring" needs to be exported by the entry point index.d.ts
```

Comment:
:eyes:

Reply:
Yeah, all of this is autogenerated from the current swagger, so we'll have warnings for now. I hope to have these all fixed by the time we hit GA.

Reply:
Went ahead and fixed